landing-page/types/alces-branding/content/styles/branding.css

```diff
@@ -18,9 +18,17 @@ body {
 .navbar .dropdown-menu {
     background-color: rgba(0, 0, 0, 0.9);
 }
+.navbar .dropdown-item:hover span {
+    color: black;
+}
 .navbar a.nav.dropdown-item:hover {
     background-color: #586775;
 }
```

Comment:
The second of these rules applies the hover styling to the account menu.  Can you remove the `.navbar .dropdown-item:hover span` rule and make the `.navbar a.nav.dropdown-item:hover` rule also match the dropdown's in the new dropdown menu.

Perhaps make the selector more generic or perhaps add a second specific selector to the rule.